src/components/Accordion/Accordion.tsx

```diff
@@ -27,15 +23,15 @@ export interface AccordionProps extends React.HTMLAttributes<HTMLDivElement> {
  *
  * [Storybook Link](Point back to yourself here)
  */
-const AccordionComponent = ({ className, type = 'single', children, ...props }: AccordionProps) => {
+const accrdionType = 'multiple';
```

Comment:
you have a typo in the const name:
```suggestion
const accordionType = 'multiple';
```